apps/frontend/src/libs/hooks/use-modal/use-modal-state-hook.ts

```diff
@@ -0,0 +1,23 @@
+import { useState } from "react";
+
+type Properties = {
+	handleCloseModal: () => void;
+	handleShowModal: () => void;
+	isOpened: boolean;
+};
+
+const useModal = (): Properties => {
+	const [isOpened, setIsOpened] = useState<boolean>(false);
+
+	const handleShowModal = (): void => {
+		setIsOpened(true);
+	};
+
+	const handleCloseModal = (): void => {
+		setIsOpened(false);
+	};
+
+	return { handleCloseModal, handleShowModal, isOpened };
```

Comment:
For hooks we use the same pattern as for components, handlers defined inside should start with handle, and handlers passed to outside should start with on. Also please unify names to have openModal/closeModal/isModalOpened instead of closeModal/showModal/isOpened
```suggestion
	return { onModalClose: handleModalClose, onModalOpen: handleModalOpen, isModalOpen };
```